### PR TITLE
fix: click events on mobile in draggable

### PR DESCRIPTION
**Pull request by j0hnfl0w**
Closes: https://github.com/snapshot-labs/sx-ui/issues/414

- fixing vuedraggable click events on mobile devices

### CODE REVIEW

**Comment by bonustrack** [comment]
@j0hnfl0w It works for me but not all the time, sometime I get locked see: 
https://user-images.githubusercontent.com/16245250/217596937-e453c8c6-afc5-4c81-86c8-ba223af1af14.mp4

**Comment by bonustrack** [comment]
Weird the same behavior is still happening for me

**Comment by j0hnfl0w** [comment]
Checked again on my Android on different browsers:
https://www.loom.com/share/fb8030e2d2684ecda63c90000eac7d77

**Comment by bonustrack** [comment]
Here is a record with latest version, for some reason when I click on a space that is already open it introduce some issue and next click failing, check: 

https://user-images.githubusercontent.com/16245250/220156924-85189e0d-cd45-4213-9494-bdd10004a7a8.mp4

Can you reproduce?

File: src/components/Sidebar.vue [hunk 1]
```diff
@@ -10,9 +12,20 @@ const spacesStore = useSpacesStore();
...
+      :touch-start-threshold="35"
+      :item-key="i => i"
+      class="space-y-3 p-2"
+    >
       <template #item="{ element }">
-        <router-link :to="{ name: 'overview', params: { id: element } }" class="block">
+        <router-link
+          :to="{ name: 'overview', params: { id: element } }"
+          class="block"
+          @click="uiStore.sidebarOpen = false"
```
Comment (Sekhmet) [inline src/components/Sidebar.vue:27]:
```suggestion
```

Do we need this change?

We do this:
https://github.com/snapshot-labs/sx-ui/blob/ecea212b7969c2467d43060fb6bd61680523aa51/src/App.vue#L39-L41

Comment (j0hnfl0w) [reply src/components/Sidebar.vue:27]: We need to handle a click on already selected space, so route watcher doesn't handle that case.

**Review by Sekhmet — APPROVED** [review]
tACK

**Pull request merged**